feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.ts

```diff
@@ -116,9 +122,18 @@ export class ConfiguratorAttributeProductCardComponent
 
   onHandleDeselect(): void {
     this.loading$.next(true);
```

Comment:
No, loading must not be emitted here! Otherwise it's e.g. not possible to change qty when the error message is visible.
Please emit loading only after you checked that the product can really be removed

Reply:
Done